dev-tools/check-shell-scripts.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -eu
+
+IFS='
+'
+
+SHELL_SCRIPTS=$(
+    find ./ \
+    -type f \
+    -not -path '*/vendor/*' \
+    -not -path './dev-tools/ci-integration.sh' \
+    -iname '*.sh'
+)
+
+# shellcheck disable=SC2086
+checkbashisms $SHELL_SCRIPTS
```

Comment:
is `"${SHELL_SCRIPTS[@]}"` valid in `sh` (i know it is in `bash`) ?

Reply:
No. There are no user-defined arrays in POSIX, there is only one quasi-array `$@` (the arguments array). That's why I had to replace all arrays with strings and use `IFS` to split them.